packages/commons/src/rate-limiter/redisRateLimiter.ts

```diff
@@ -111,15 +145,17 @@ export async function initRedisRateLimiter(config: {
   ): Promise<number> {
     return redisClient
       .get(`${config.limiterGroup}:${organizationId}`)
-      .then(Number);
+      .then(Number)
+      .catch(() => 0);
```

Comment:
Let's first get this from the insurance limiter, then we fallback to 0 otherwise
```suggestion
      .catch(() =>
        insuranceLimiter
          .get(organizationId)
          .then((res) => res?.consumedPoints || 0)
      );
```